java/com/facebook/crypto/streams/NativeGCMCipherInputStream.java

```diff
@@ -136,6 +136,6 @@ public synchronized void reset() throws IOException {
 
   @Override
   public long skip(long byteCount) throws IOException {
-    throw new UnsupportedOperationException();
+    return super.skip(byteCount);
```

Comment:
i would rather avoid creation of the 2048 byte buffer if possible to be as memory efficient as we can.